docker-compose.yml

```diff
@@ -1,103 +1,117 @@
-version: '3'
+version: '3.4'
+
+x-server-config: &server-config
+  METRICS_SERVER: "http://host.docker.internal:8001"
+  README_API_KEY: rdme_abcdefghijklmnopqrstuvwxyz
+
+x-extra_hosts: &default-extra_hosts
+  # Docker in Github doesn't automaticaly map these and without `host.docker.internal`
+  # working our SDK HTTP servers can't talk to our mock Metrics server.
+  - "host.docker.internal:host-gateway"
```

Comment:
Lost most of the day to this. 🫠

Reply:
😭 glad you figured it out. Networking in Docker is still a bit of a 🤯 to me.